filebeat/prospector/prospector_log.go

```diff
@@ -112,17 +114,69 @@ func (l *Log) Run() {
 	}
 }
 
+// globPattern detects the use of "**" and expands it to standard glob patterns up to a max depth
+func (l *Log) globPatterns(pattern string) ([]string, error) {
+	if l.config.DoubleWildcardMaxDepth == 0 {
+		return []string{pattern}, nil
+	}
+	var patterns []string
+	isAbs := filepath.IsAbs(pattern)
+	patternList := strings.Split(pattern, "/")
+	for i, dir := range patternList {
+		if len(patterns) > 0 {
+			if dir == "**" {
+				err := fmt.Sprintf("glob(%s) failed: cannot specify multiple ** within a pattern", pattern)
+				logp.Err(err)
+				return nil, errors.New(err)
+			}
+			for i := range patterns {
+				patterns[i] = filepath.Join(patterns[i], dir)
+			}
+		} else if dir == "**" {
+			prefix := filepath.Join(patternList[:i]...)
+			if isAbs {
+				prefix = "/" + prefix
+			}
+			wildcards := ""
+			for j := uint8(0); j < l.config.DoubleWildcardMaxDepth; j++ {
+				wildcards = filepath.Join(wildcards, "*")
+				patterns = append(patterns, filepath.Join(prefix, wildcards))
+			}
+		}
+	}
+	if len(patterns) == 0 {
+		patterns = []string{pattern}
```

Comment:
I'd say no, since for example, I use `""` in the filebeat modules to disable a fileset on a particular version.